### PR TITLE
[Test Hackathon] Fix SendIntegrationTestCase

### PR DESCRIPTION
**Pull request by arunans23**
## Purpose
Fix SendIntegrationTestCase as part of Test Hackathon

### CODE REVIEW

File: integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/config/test_sequences_config/receivingSequence_Conf.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!--
+  ~  Copyright (c) 2005-2019, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```
Comment (chanikag) [inline integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/config/test_sequences_config/receivingSequence_Conf.xml]: License header has to be corrected

File: integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/governance/test_sequences_gov/receivingSequence_Gov.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<!--
+  ~  Copyright (c) 2005-2019, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```
Comment (chanikag) [inline integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/governance/test_sequences_gov/receivingSequence_Gov.xml]: License header has to be corrected

File: integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/local/test_sequences_local/receivingSequence_Local.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<!--
+  ~  Copyright (c) 2005-2019, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```
Comment (chanikag) [inline integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/local/test_sequences_local/receivingSequence_Local.xml]: License header has to be corrected

Comment (arunans23) [reply integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/local/test_sequences_local/receivingSequence_Local.xml]: done

File: integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/governance/test_sequences_gov/receivingSequence_Gov.xml (hunk 2 above)
Comment (arunans23) [reply integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/governance/test_sequences_gov/receivingSequence_Gov.xml]: done

File: integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/config/test_sequences_config/receivingSequence_Conf.xml (hunk 1 above)
Comment (arunans23) [reply integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/config/test_sequences_config/receivingSequence_Conf.xml]: done

**Pull request merged**